canonicalwebteam/discourse/parsers/category.py

```diff
@@ -14,7 +14,6 @@ class CategoryParser(BaseParser):
     """
 
     def __init__(self, api, index_topic_id, url_prefix):
-        self.category_metadata = None
         return super().__init__(api, index_topic_id, url_prefix)
 
     def parse_index_topic(self):
```

Comment:
I would advise on to adding return types to functions, like here you changed the return type from `none` to `dictionary` , it this is then used by another dev, they could mistakenly invoke it as if it was assigning and returning none.

Reply:
Good idea! Maybe we should start enforcing this through our linter/checks

Reply:
I am actually going to omit this for now and include it in the test writing portion of this project.